pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -479,33 +479,61 @@ func initializeSelfPublishHandles(ps *pubsub.PubSub, ctx *vaultMgrContext) {
 	ctx.pubVaultStatus = pubVaultStatus
 }
 
-//GetOperInfo gets the current operational state of fscrypt. (Deprecated)
-func GetOperInfo() (info.DataSecAtRestStatus, string) {
+//getFscryptOperInfo does some basic fscrypt checks
```

Comment:
I still just returns the info, hence the "checks" part of the comment seems misleading.

Reply:
changed it